swupd-overdue.c

```diff
@@ -31,12 +31,41 @@
 #include <time.h>
 #include <string.h>
 #include <errno.h>
+#include <systemd/sd-bus.h>
 
 /* What file do we check against? */
 #define DATESTAMPFILE "/usr/share/clear/versionstamp"
 /* Default threshold is 14 days */
 #define THRESHOLD 86400 * 14
 
+int systemd_start(char *service)
```

Comment:
Since this code is only executed once, it shouldn't be a function, and we should just include the whole section of code inside `main()` verbatim